mail/mailsend/patches/0002-Removed-API-deprecated-by-OpenSSL-1.1.0.patch

```diff
@@ -39,6 +40,36 @@ index d275e6a..51bfb1d 100755
  #endif /* ! HAVE_OPENSSL */
                  rc = 0;
                  goto ExitProcessing;
+diff --git a/utils.c b/utils.c
+index abfb86e..8b02a26 100755
+--- a/utils.c
++++ b/utils.c
+@@ -440,10 +440,14 @@ void initialize_openssl(char *cipher)
+     static const char rnd_seed[]="my huge entropy for rng.. blah";
+     SSL_CTX *ssl_ctx=(SSL_CTX *) NULL;
+     SSL *ssl=NULL;
++#if OPENSSL_VERSION_NUMBER < 0x10100000L
+     SSL_library_init();
+     SSL_load_error_strings();
+-    RAND_seed(rnd_seed,sizeof(rnd_seed));
+     OpenSSL_add_all_algorithms();
++#else
++    OPENSSL_init_ssl(0, NULL);
```

Comment:
I wouldn't make this call, as the default init is done by the library automatically.